.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -221,62 +188,54 @@ jobs:
         env:
           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN_AIRBYTE_TEAM }}
 
-  publish-docker-image:
-    timeout-minutes: 240
+  bump-manifest-source:
+    name: Bump CDK dependency of source-declarative-manifest
     needs:
-      - start-publish-docker-image-runner-0
-      - publish-cdk
-    runs-on: runner-pool-${{ github.run_id }}
+    - bump-version
+    - publish-cdk
+    runs-on: ubuntu-latest
     steps:
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install Poetry
+        id: install_poetry
+        uses: snok/install-poetry@v1
       - name: Checkout Airbyte
         uses: actions/checkout@v3
         with:
-          repository: airbytehq/airbyte
+          repository: ${{ github.event.inputs.repo }}
           ref: ${{ github.event.inputs.gitref }}
-      - name: Install Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: "3.10"
-      - name: Publish source-declarative-manifest
-        id: publish
-        env:
-          DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
-          DOCKER_HUB_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
-          # Oracle expects this variable to be set. Although usually present, this is not set by default on Github virtual runners.
-          TZ: UTC
-        uses: Wandalen/wretry.action@v1.0.42
+      - name: Bump CDK dependency of source-declarative-manifest
+        run: |
+          cd airbyte-integrations/connectors/source-declarative-manifest
+          poetry add airbyte-cdk==${{needs.bump-version.outputs.new_cdk_version}}
+      - name: Bump version of source-declarative-manifest
+        uses: ./.github/actions/run-airbyte-ci
         with:
-          command: |
-            docker login -u ${DOCKER_HUB_USERNAME} -p ${DOCKER_HUB_PASSWORD}
-            ./tools/integrations/manage.sh publish airbyte-cdk/python false
-      - name: Post failure to Slack channel dev-connectors-extensibility
-        if: ${{ failure() }}
-        uses: slackapi/slack-github-action@v1.23.0
-        continue-on-error: true
+          context: "master" # TODO change to ref?
```

Comment:
I'm not sure if this even matters here because I don't think `bump_version` cares about context - checking into this. Not sure if we want to pass everything in to this like we do most places I think, or remove the unneeded inputs to avoid confusion (e.g. python registry token is irrelevant as we're not pushing to pypi). I'll probably test out which ones are unnecessary and remove them.

Reply:
I think it might be useful to use ref for troubleshooting purposes, but I also think we should just remove the parameter if we're unsure it'll be safe / respected